src/components/controls/dates/DateRangePicker/DateRangePicker.tsx

```diff
@@ -43,6 +43,9 @@ interface DateRangePickerBaseProps {
 
   /** Props passed to root element. */
   wrapperProps?: BoxProps
+
+  /** dayjs input format */
```

Comment:
What do you think about noting the default here as well?

```suggestion
  /** `dayjs` input format. Default is `YYYY-MM-DD`. */
```